open_api_framework/conf/base.py

```diff
@@ -211,6 +211,7 @@
     "csp",
     "corsheaders",
     "vng_api_common",
+    "vng_api_common.authorizations",
```

Comment:
@SonnyBA come to think of it, I'm actually not sure if all components use this (e.g. Objects/Objecttypes), so wouldn't it be better to add this only to the components that need it? :thinking:

Reply:
I think it would be best to indeed move the middleware to the authorizations app. I think this also relates to https://github.com/maykinmedia/open-api-framework/issues/67, we should make these dependencies should be optional at some point probably